routes/index.js

```diff
@@ -48,14 +48,17 @@ var sendEmail = function(recipient, attachments, duration) {
   return Promise.all(attachmentPromises).then(function(attachments) {
     var fromDate = moment(duration.fromDate).format('DD-MMM-YYYY');
     var toDate = moment(duration.toDate).format('DD-MMM-YYYY');
-    var subject = inflection.capitalize(duration.title) + ' Stats: ' + fromDate + ' to ' + toDate
+    var subject = inflection.capitalize(duration.title) + ' Stats: ' + fromDate + ' to ' + toDate;
     var data = {
       from: process.env.CM_EMAIL_SENDER,
       to: recipient || process.env.CM_EMAIL_RECIPIENT,
       subject: subject,
       text: 'Developer statistics for the given period',
       attachment: attachments
     };
+    if(typeof ccRecipient != 'undefined'){
```

Comment:
added default email...
add in heroku config
`CM_EMAIL_CC_RECIPIENT = fahad@caremerge.com`